lib/src/kvs/ds.rs

```diff
@@ -1292,3 +1383,78 @@ impl Datastore {
 		Ok(())
 	}
 }
+
+async fn catchup_live_queries(
+	mut tx: Transaction,
+	local_live_queries: Arc<RwLock<BTreeMap<LqIndexKey, LqIndexValue>>>,
+	notification_channel_sender: Sender<Notification>,
+	lqs_to_update: Vec<(LqIndexKey, LqIndexValue)>,
+	vs: Versionstamp,
+) {
+	for (k, v) in lqs_to_update {
+		// Process notifications
+		let changes = {
+			let changes = cf::read(
+				&mut tx,
+				&k.ns,
+				&k.db,
+				Some(&k.tb),
+				ShowSince::Versionstamp(conv::versionstamp_to_u64(&v.vs)),
+				Some(1000),
```

Comment:
Shouldn't we put this in a constant so it's clear what it does?

Reply:
Declared temporary const